packages/components/src/templates/next/render/renderComponentPreviewText.ts

```diff
@@ -98,6 +99,8 @@ export function renderComponentPreviewText({
       return getTextContentOfProse(component.content)
     case "video":
       return component.title || "Video embed"
+    case DYNAMIC_STATISTICS_TYPE:
+      return component.apiEndpoint
```

Comment:
don't know what's the best preview text to display - will use `apiEndpoint` now since it's a required field + we aren't supporting this on studio yet